Extensions/RepeatEveryXSeconds-header.json

```diff
@@ -0,0 +1,15 @@
+{
+  "shortDescription": "This allows to trigger an event every X seconds.",
+  "extensionNamespace": "",
+  "fullName": "Repeat every X seconds",
+  "name": "RepeatEveryXSeconds",
+  "version": "0.1.0",
+  "url": "Extensions/RepeatEveryXSeconds.json",
+  "headerUrl": "Extensions/RepeatEveryXSeconds-header.json",
+  "tags": "repeat,timer,time,convenience",
+  "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/repeat.svg",
+  "eventsBasedBehaviorsCount": 1,
+  "eventsFunctionsCount": 2,
+  "description": "Convenience functions for timers to trigger a condition every X seconds.",
```

Comment:
Not "functions" => "conditions, actions and behaviors".

Remember, the term function should not be surfaced outside of an extension, for users the concepts to understand are "conditions", "actions" and "behaviors" :)